Docs--Tag-link-update (#1541)
carbon-design-system/carbon-for-ibm-dotcom-website: src/pages/components/tags.mdx

---
title: Tags
description: Tags help label and categorize content using keywords. Tag link enables a tag to be clickable and navigate users to related content.
---

import ComponentDescription from 'components/ComponentDescription';
import ComponentFeedback from 'components/ComponentFeedback';
import ResourceLinks from 'components/ResourceLinks';

<ComponentDescription name="Tags" type="ui" />

<AnchorLinks>

<AnchorLink>Resources</AnchorLink>
<AnchorLink>Overview</AnchorLink>
<AnchorLink>Tag</AnchorLink>
<AnchorLink>Tag link</AnchorLink>
<AnchorLink>Tag group</AnchorLink>
<AnchorLink>Feedback</AnchorLink>
  
</AnchorLinks>

## Overview

TBD content

## Tag

> The Carbon tag is primarily used inside the [Card](https://www.ibm.com/standards/carbon/components/cards) component in Carbon for IBM.com. This enables Cards to display labels that categorize the Card's contents or highlight aspects of the content.

Reviewer: ```suggestion
The Carbon tag is primarily used inside the [card](https://www.ibm.com/standards/carbon/components/cards) component in Carbon for IBM.com. This enables cards to display labels that categorize the card's contents or highlight aspects of the content.
```

Shouldn't be title-cased, right?

> For more information on Tag, visit the [Tag guidance](https://carbondesignsystem.com/components/tag/usage/) for the Carbon Design System.

Reviewer: ```suggestion
For more information, visit Carbon Design System's tag [guidance](https://carbondesignsystem.com/components/tag/usage/) page.
```

## Tag link

> Tag link is a clickable element that links the topic to another page for users to find more information about the topic. It uses Carbon's [Tag](https://carbondesignsystem.com/components/tag/usage/) as a base, and uses a larger type token and additional spacing around the label. Unlike Tag, Tag link has limited color styles for the label and background.

Reviewer: ```suggestion
Tag link is a clickable element that links the topic to another page for users to find more information about the topic. It uses Carbon's [tag](https://carbondesignsystem.com/components/tag/usage/) as a base, and uses a larger type token and additional spacing around the label. Unlike `tag`, `tag link` has limited color styles for the label and background.
```

> Tag link is primarily used inside of a Tag group (see more below).

Reviewer: ```suggestion
Tag link is primarily used inside of a tag group (see more below).
```

<ResourceLinks name="Tag link" type="ui" multiComponent />

### Content guidance for Tag link

| Element | Content type | Required | Instances | Character limit <br/>(English / translated) | Notes               |
| ------- | ------------ | -------- | --------- | ------------------------------------------- | ------------------- |
| Label   | Text         | No       | 1         | 25 / 35                                     | URL added in setup. |

## Tag group

> Tag group is a collection of Tag links that can be added to the [Lead space](https://www.ibm.com/standards/carbon/components/leadspace). Tag group adds additional context to the Lead space content, and each Tag link can navigate users to a broader category within IBM.com.

Reviewer: ```suggestion
Tag group is a collection of tag links that can be added to the [lead space](https://www.ibm.com/standards/carbon/components/leadspace). Tag group adds additional context to the lead space content, and each tag link can navigate users to a broader category within IBM.com.
```

---
title: Tags
description: Tags help label and categorize content using keywords. Tag link enables a tag to be clickable and navigate users to related content.
---

import ComponentDescription from 'components/ComponentDescription';
import ComponentFeedback from 'components/ComponentFeedback';
import ResourceLinks from 'components/ResourceLinks';

<ComponentDescription name="Tags" type="ui" />

<AnchorLinks>

<AnchorLink>Resources</AnchorLink>
<AnchorLink>Overview</AnchorLink>
<AnchorLink>Tag</AnchorLink>
<AnchorLink>Tag link</AnchorLink>
<AnchorLink>Tag group</AnchorLink>
<AnchorLink>Feedback</AnchorLink>
  
</AnchorLinks>

## Overview

> Tags are elements that add additional context to content that it appears in. In Carbon for IBM.com, the Carbon tag is used directly in the card component. Tag link is a clickable version of the Carbon tag that is often used inside a tag group.

Reviewer 2: ```suggestion
Tags are used to add additional context to, or categorize content that they appear in. In Carbon for IBM.com, the Carbon tag is used directly in the card component. Tag link is a clickable version of the Carbon tag that is often used inside a tag group to represent poly-hierarchy.
```
Suggestion to remove "element"
